test/integration/tests/button/eligibility.js

```diff
@@ -235,3 +238,56 @@ describe('venmo button eligibility', () => {
         }
     });
 });
+
+describe('paypal on file eligibility', () => {
+    beforeEach(() => {
+        createTestContainer();
+    });
+
+    afterEach(() => {
+        destroyTestContainer();
+    });
+
+    it(`should render paypal button with wallet instrument labels when ppof eligibility is true`, (done) => {
+        const fundingSource = FUNDING.PAYPAL;
+        const content = testContent;
+        const wallet = {
+            [fundingSource]: {
+                instruments: [
+                    {
+                        accessToken: null,
+                        instrumentID: 'abc12345',
+                        label: '••1234',
+                        logoUrl: null,
+                        oneClick: true,
+                        planID: null,
+                        secondaryInstruments: [{
+                            instrumentID: "BALANCEUSD",
+                            label: "PayPal Balance",
+                            type: "BALANCE"
+                        }],
+                        tokenID: null,
+                        type:    'card',
+                        vendor:  'VISA'
+                    }
+                ]
+            }
+        };
+
+        window.paypal.Buttons({
+            content,
+            fundingSource,
+            wallet,
+            showPayLabel: false,
+            test:   {
+                onRender: () => {
+                    setTimeout(() => {
+                        assert.equal(getElementRecursive('.balance .paypal-button-text').innerHTML, 'Balance &amp;');
+                        assert.equal(getElementRecursive('.fi-label .paypal-button-text').innerHTML, '••1234');
+                        done();
+                    }, 1000);
```

Comment:
Ideally we'd like to remove the `setTimeout()` to avoid the tests running slow. Do the tests still pass if you remove the `setTimeout()`? If not, can you try reducing the time to `100` ms?

Reply:
The test fails if I completely remove `setTimeout()`. I'll reduce the time to 100ms and will push the change if it passes.

Reply:
I take that back. I can remove `setTimeout()` as long as I still call `done()`. I just pushed the change. 😄